pkgs/applications/audio/hushboard/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, wrapGAppsHook
```

Comment:
Put these lists in alphabetical order, except for the three first lines. It helps reading the code.

Do the same on the other lists as well.